evm/src/cpu/stack.rs

```diff
@@ -53,6 +53,20 @@ pub(crate) const MLOAD_GENERAL_OP: Option<StackBehavior> = Some(StackBehavior {
     disable_other_channels: false,
 });
 
+pub(crate) const KECCAK_GENERAL_OP: Option<StackBehavior> = Some(StackBehavior {
+    num_pops: 4,
+    pushes: true,
+    new_top_stack_channel: Some(NUM_GP_CHANNELS - 1),
+    disable_other_channels: true,
+});
+
+pub(crate) const JUMPDEST_OP: Option<StackBehavior> = Some(StackBehavior {
+    num_pops: 0,
+    pushes: false,
+    new_top_stack_channel: None,
+    disable_other_channels: true,
+});
+
```

Comment:
We never exploit that those are wrapped inside an `Option`, as we unwrap them directly, so I'd just change their type to simply `StackBehavior`.
We could also apply this to the others, either in this PR or in a follow-up one.